src/Features/LanguageServer/Protocol/Handler/Symbols/DocumentSymbolsHandler.cs

```diff
@@ -30,17 +30,17 @@ public DocumentSymbolsHandler()
         }
 
         public async Task<object[]> HandleRequestAsync(Solution solution, DocumentSymbolParams request,
-            ClientCapabilities clientCapabilities, CancellationToken cancellationToken)
+            ClientCapabilities clientCapabilities, string clientName, CancellationToken cancellationToken)
```

Comment:
```suggestion
            ClientCapabilities clientCapabilities, string? clientName, CancellationToken cancellationToken)
```

Reply:
And this one escaped `#nullable`?

Reply:
This one is one of the more tricky ones.  I'll do it in a followup to enable nullable on the entire project.